tests/tests/swfs/avm2/netconnection_send_remote/test.toml

```diff
@@ -0,0 +1,4 @@
+# Note that this test doesn't run natively in Flash Player, see README.md
+
+num_ticks = 10
+log_fetch = true
```

Comment:
This field is not documented in `tests/README.md`? (We probably should do that)

Reply:
Opened #13909 for that